source/stylesheets/base/_reboot.scss

```diff
@@ -1,3 +1,21 @@
 *, :before, :after {
   box-sizing: border-box;
 }
+
+@media print {
```

Comment:
If we're going all in on organisational efforts for the CSS, should these (and `#sidebar-wrapper @media print { ... }`) live in a `_print` file?

Reply:
I moved these closer to their parent definitions :) I'm not so keen on having a `_print` file, historically I've ended up forgetting to update them.

Reply:
Fair enough - keeping them with the parent seems like a good compromise.